sdk/communication/communication-chat/src/chatClient.ts

```diff
@@ -75,10 +67,6 @@ export class ChatClient {
   ) {
     this.tokenCredential = credential;
     this.clientOptions = { ...options };
-    this.clientOptions.signalingClientOptions = {
```

Comment:
This should be kept; using a local type definition ensures that we have type-checking and can avoid the use of `any`